templates/terraform/modify_delete_url.erb

```diff
@@ -0,0 +1,4 @@
+url, err = replaceVars(d, config, "<%= object.__product.default_version.base_url -%>{{name}}")
```

Comment:
Consider adding a comment here with an explanation about why you added this block. I would take the explanation you added in this PR description. The generated code is a bit confusing to read the way it is.

Reply:
Another option is adding a `delete_url` and `create_url` to the Resource class.
If set `delete_url` and `create_url` are nil, default to the `base_url`.
Update the template to use these new urls.